plugin-build/src/main/kotlin/io/sentry/android/gradle/SentryPluginExtension.kt

```diff
@@ -47,4 +47,13 @@ abstract class SentryPluginExtension @Inject constructor(project: Project) {
     /** List of Android build flavors that should be ignored by the Sentry plugin. */
     val ignoredFlavors: ListProperty<String> = objects.listProperty(String::class.java)
         .convention(emptyList())
+
+    /**
+     * Forces dependencies instrumentation, even if they were already instrumented.
+     * Useful when there are issues with code instrumentation, e.g. the dependencies are
+     * partially instrumented.
+     * Defaults to false.
+     */
+    val forceInstrumentDependencies: Property<Boolean> = objects.property(Boolean::class.java)
```

Comment:
I am not sure if we should expose this to the end users, as it only makes sense for devs of the plugin (from my understanding). But I'd probably tackle this later, when we figure out what configuration should we provide to the users